src/EditorFeatures/Core.Wpf/NavigateTo/NavigateToItemProvider.Callback.cs

```diff
@@ -39,9 +43,41 @@ public void Done(bool isFullyLoaded)
                 }
             }
 
-            public Task AddItemAsync(Project project, INavigateToSearchResult result, CancellationToken cancellationToken)
+            public Task AddResultsAsync(ImmutableArray<INavigateToSearchResult> results, CancellationToken cancellationToken)
```

Comment:
1. naming become more consistent.  There is AddResultsAsync for when there's an array of INavigateToSearchResult, and AddItemsAsync for when it's an array of RoslynNavigateToItem.
2. These callbacks now take arrays so the server can send a lot at once when it has many items available (if the searchign threads are faster than the sending thread).